distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1422,7 +1422,7 @@ public void testValidationChecker() throws Exception {
         // Configure invalid filtering
         configureMetacardValidityFilterPlugin(Arrays.asList(""));
 
-        configureShowInvalidMetacards("true");
```

Comment:
I think you'll need to pay attention to updating these method calls in all DDF-based projects as well.